docs/root/mithril/mithril-protocol/protocol.md

```diff
@@ -3,126 +3,119 @@ sidebar_position: 1
-the protocol in layman’s terms.
+The full details of the Mithril protocol are provided in the published paper. This section is a high-level description of
+the protocol.
 
-In a nutshell, Mithril is a signature scheme that generates a certificate that convinces
-the verifiers that a portion of the stake of a system has signed a message.
+In a nutshell, Mithril is a signature scheme that generates a certificate to convince verifiers that a portion of the stake of a system has signed a message.
 
-However, rather than taking the whole set of
-stake holders, Mithril
-"randomly selects" a subset of them, and requires a portion of that subset to provide a signature.
+However, rather than taking the whole set of stakeholders, Mithril
+'randomly' selects a subset of them, and requires a portion of that subset to provide a signature.
 
-An analogy would be
-that Mithril signers participate in a lottery which defines if they are entitled to sign a particular message. In case
-they win this lottery, then they can go ahead and sign the message. Whenever a sufficient number of parties have won the
-lottery (and submitted their respective signatures), a certificate can be computed.
+Mithril signers participate in a lottery that defines if they are entitled to sign a particular message.
+The lottery winners can then sign the message. When there are enough signatures, a certificate can be computed.
 
-## The phases of the protocol
+## Protocol phases
 
-The protocol has 3 different phases:
+The protocol has three phases:
 
-1. **Protocol Establishment phase** which determines the parameters that will be used by Mithril participants. Also,
-   during this phase, the set of "allowed signers" is defined.
-2. **Initialization phase** during which Mithril nodes generate and exchange keys,
+1. **Protocol establishment phase** during which parameters are defined for Mithril participants and a set of *allowed signers* is defined
+2. **Initialization phase** during which Mithril nodes generate and exchange keys
 3. **Operations phase** during which nodes sign and aggregate signatures of messages to produce certificates.
 
 :::note Note
 
-Note that all 3 phases require the set of parties (`P` in the paper) to be fixed. During the protocol establishment phase
+Note that all three phases require a set of parties (`P` in the paper) to be fixed. During the protocol establishment phase
 three important parameters are generated:
 
-* `m`, defines the number of "lotteries" that a single user can participate in to sign a message.
-* `k`, defines the required number of single signatures to produce a valid certificate.
-* `phi_f` (as denoted in the library) which can be interpreted as the chance of a signer to win a lottery.
+* `m`: defines the number of lotteries that a single user can participate in to sign a message
+* `k`: defines the required number of signatures for a valid certificate
```

Comment:
Is it still accurate as we send all the won lotteries indices along with one message signature now @iquerejeta @curiecrypt ?

Reply:
Yes, it seems to me that it is still correct. Even if we send more than `k` signatures in the certificate, that is still the security parameter that defines the minimal number of signatures. i.e. at least `k` signatures are need to generate a valid certificate.